gfx.cpp

```diff
@@ -8200,8 +8179,9 @@ class GfxInternal
                 for(size_t i = 0; i < kernel.exports_.size(); ++i)
                     max_export_length = GFX_MAX(max_export_length, strlen(kernel.exports_[i].c_str()));
                 max_export_length += 1;
-                char *lib_export = (char *)alloca(max_export_length);
-                WCHAR *wexport = (WCHAR *)alloca(max_export_length << 1);
+                char lib_export[1024];
+                WCHAR wexport[1024];
+                GFX_ASSERT(max_export_length < 1024);
```

Comment:
Same statement than before, maybe we should just use `gfxMalloc()` here?
This might actually be related to this issue now that I think of it 🙂 https://github.com/Radeon-Pro/Capsaicin/issues/567